### PR TITLE
New package semanticversion-py + Django updates

### PR DESCRIPTION
**Pull request by dhomeier**
Packaged [semantic_version](https://pypi.org/project/semantic_version/), a Python library implementing the 'SemVer' scheme, in preparation for another project. This provides some Django support and requires the latter for testing, so I've added a `django-py37` type to it and also updated to LTS version 1.11.18 for py27+py34, and to the latest upstream 2.1.5 for py35+. Finally updated `django-localflavor-py` to 2.1  and added tests for all (the Django tests only fully succeed with Python 3.7).

### CODE REVIEW

**Comment by dhomeier** [comment]
Both `django-1.11.18` and 2.1.5 should support `geos-3.6`, but I've only updated the 2.1.5 package. Just checked that the LTS builds and passes the same tests with `Type: geos(3.6.1)`.

File: 10.9-libcxx/stable/main/finkinfo/web/django-localflavor-py.info [hunk 1]
```diff
@@ -45,8 +60,16 @@ For a full list of available localflavors, see
 http://django-localflavor.readthedocs.org/
 <<
 
+DescPackaging:<<
+Test results could depend on installed django; alternative would be to enforce
+a specific django version (conditional on %type_pkg[python]) in TestDepends.
+Included coverage and invoke in TestDepends, but tox will do a sandbox install
```
Comment (nieder) [inline 10.9-libcxx/stable/main/finkinfo/web/django-localflavor-py.info]: The tox test command is just `invoke test`, so could easily modify TestScript to just call the right flavor of invoke-pyXX directly and not worry about tox's sandbox downloading.

Comment (dhomeier) [reply 10.9-libcxx/stable/main/finkinfo/web/django-localflavor-py.info]: Thanks for the pointer; I think I had given up last time I tried to understand fox's inner workings...
But it seems the requirement `invoke<0.12` is a real one; running Fink's invoke 1.0.0  directly fails immediately with a `TypeError: Tasks must have an initial Context argument!`.
I've built a 0.11.1 version of the invoke-py package, which runs the test successfully, but is it worth maintaining an additional versioned invoke for that?

Comment (dhomeier) [reply 10.9-libcxx/stable/main/finkinfo/web/django-localflavor-py.info]: OK, found out how to update the tasks definitions to accommodate the `Context` argument. Tested also with the invoke update to 1.2.0 from #240.

**Comment by kamischi** [comment]
I wanted to check on 10.14, but gcc5 and gcc7 are still missing.

File: 10.9-libcxx/stable/main/finkinfo/web/django-py.info [hunk 2]
```diff
@@ -16,32 +16,55 @@ BuildDepends: setuptools-tng-py%type_pkg[python]
...
+    pytz-py%type_pkg[python],
+    yaml-py%type_pkg[python]
+  <<
+  TestScript: <<
+    #!/bin/sh -ev
+    cd tests
+    # Run tests in parallel on all available cores (default) - set to 1 for better diagnostics
+    # ~25 ascii/unicode errors for Python < 3.7
+    if [ %type_pkg[python] -lt 37 ]; then
+      PYTHONPATH=%b/build/lib %p/bin/python%type_raw[python] -B runtests.py --parallel=0 || exit 1
```
Comment (nieder) [inline 10.9-libcxx/stable/main/finkinfo/web/django-py.info]: Setting `LANG="en_US.UTF-8" ` for the py36 and below tests makes them pass for me (fixes the ascii encoding failure).

**Comment by nieder** [comment]
django-localflavor-py27 to py36 are failing for me like this:
````
/sw/bin/invoke-py35 test || exit 2
Python version: 3.5.6 (default, Jan 12 2019, 05:22:57) 
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)]
Traceback (most recent call last):
  File "/sw/bin/django-admin.py", line 11, in <module>
    load_entry_point('Django==2.1.5', 'console_scripts', 'django-admin')()
  File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 484, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 476, in get_distribution
    dist = get_provider(dist)
  File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 352, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 895, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 781, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'Django==2.1.5' distribution was not found and is required by the application
Coverage.py warning: No data was collected. (no-data-collected)
````
(the py27 version complains about Django==1.11.18 instead). I have the current versions from this pull request installed. django-localflavor-py37 builds and passes tests.
````
 i   django-py27             1.11.18-1         Python web framework
     django-py34             1.11.18-1         Python web framework
 i   django-py35             2.1.5-1           Python web framework
 i   django-py36             2.1.5-1           Python web framework
 i   django-py37             2.1.5-1           Python web framework
````

**Comment by nieder** [comment]
semanticversion-pyXX passes tests here.

**Comment by dhomeier** [comment]
@nieder, I cannot reproduce the test failures for the py27-py36 variants, although I also have `/sw/bin/django-admin.py` installed and pointing to `/sw/bin/django-admin3.7`. Which invoke version have you been using?

Updated both variants to the latest upstream versions, the `django-localflavor-py` tests still pass here for both.

**Comment by nieder** [comment]
And now django-localflavor-py* pass tests for me.
Testing django...

**Comment by nieder** [comment]
> django-localflavor-py27 to py36 are failing for me like this:
> 
> ```
> /sw/bin/invoke-py35 test || exit 2
> Python version: 3.5.6 (default, Jan 12 2019, 05:22:57) 
> [GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)]
> Traceback (most recent call last):
>   File "/sw/bin/django-admin.py", line 11, in <module>
>     load_entry_point('Django==2.1.5', 'console_scripts', 'django-admin')()
>   File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 484, in load_entry_point
>     return get_distribution(dist).load_entry_point(group, name)
>   File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 476, in get_distribution
>     dist = get_provider(dist)
>   File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 352, in get_provider
>     return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
>   File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 895, in require
>     needed = self.resolve(parse_requirements(requirements))
>   File "/sw/lib/python3.7/site-packages/pkg_resources/__init__.py", line 781, in resolve
>     raise DistributionNotFound(req, requirers)
> pkg_resources.DistributionNotFound: The 'Django==2.1.5' distribution was not found and is required by the application
> Coverage.py warning: No data was collected. (no-data-collected)
> ```
> 
> (the py27 version complains about Django==1.11.18 instead). I have the current versions from this pull request installed. django-localflavor-py37 builds and passes tests.

I think the problem here is that django-localflavor-pyXX calls the unversioned /sw/bin/django-admin.py, which has a high probability of not being the same py version. In this example py35 is using py37.
I just purged all my py34 mods, then failed to rebuild django-localflavor-py34 (it was using files belonging to setuptools-tng-py37 according to the error, just like above). But if I remove django-py37, then the unversioned /sw/bin/django-admin.py points to the py36 copy, and the django-localflavor-py34 fails pointing to the same setuptools-tng-py37 files as above.
Removing setuptools-tng-py37 complains about:
````
  File "/sw/bin/coverage", line 6, in <module>
    from pkg_resources import load_entry_point
ModuleNotFoundError: No module named 'pkg_resources'
````
Further removing coverage-py37 lets django-localflavor-py34 build.
Putting setuptools-tng-py37 back (but not coverage-py37) lets django-localflavor-py34 build.

So there's some weird games going on for what non-versioned files are being pulled in (coverage-py is the culprit?).

**Comment by nieder** [comment]
tasks.py in django-localflavor has to be edited so that it runs the right pyvariant of coverage and of django-admin.py.

**Comment by kamischi** [comment]
semanticversion needs a depends on setuptools-tng.

**Comment by dhomeier** [comment]
> tasks.py in django-localflavor has to be edited so that it runs the right pyvariant of coverage and of django-admin.py.

Strange, I could not reproduce those errors even after removing `django-localflavor-py37`, `coverage-py37` and even `invoke-py37` (one by one)...
But modifying `tasks.py` to
```
test_cmd = 'coverage%type_raw[python] run `which django-admin%type_raw[python]` test --settings=tests.settings'
```
at least lets the tests still pass on my system.

**Comment by nieder** [comment]
Where you trying to build django-localflavor-py34 with both coverage-py37 and setuptools.tng-py37 installed? That's a consistent fail for me.

For the fix, I would take out the `which ...` part, and just hardcode the full path to the right /sw/bin/django-admin.pyX.Y. Less chance of accidentally picking up something incorrectly.

**Comment by dhomeier** [comment]
That was step two, nothing suspicious here:
```
 i   coverage-py27                 4.5.1-1                Python code coverage for testing
 i   coverage-py34                 4.5.1-1                Python code coverage for testing
 i   coverage-py35                 4.5.1-1                Python code coverage for testing
 i   coverage-py36                 4.5.1-1                Python code coverage for testing
 i   coverage-py37                 4.5.1-1                Python code coverage for testing
 i   django-gis-py27               1.11.20-1              GIS/geo contrib application for django
     django-gis-py34               1.11.20-1              GIS/geo contrib application for django
     django-gis-py35               2.1.7-1                GIS/geo contrib application for django
     django-gis-py36               2.1.7-1                GIS/geo contrib application for django
     django-gis-py37               2.1.7-1                GIS/geo contrib application for django
 i   django-localflavor-py27       2.1-1                  Country-specific Django helpers
 i   django-localflavor-py34       2.1-1                  Country-specific Django helpers
     django-localflavor-py35       2.1-1                  Country-specific Django helpers
 i   django-localflavor-py36       2.1-1                  Country-specific Django helpers
     django-localflavor-py37       2.1-1                  Country-specific Django helpers
 i   django-py27                   1.11.20-1              Python web framework
 i   django-py34                   1.11.20-1              Python web framework
 i   django-py35                   2.1.7-1                Python web framework
 i   django-py36                   2.1.7-1                Python web framework
 i   django-py37                   2.1.7-1                Python web framework
 i   setuptools-tng-py27           40.2.0-1               EasyInstall and python eggs
 i   setuptools-tng-py34           40.2.0-1               EasyInstall and python eggs
 i   setuptools-tng-py35           40.2.0-1               EasyInstall and python eggs
 i   setuptools-tng-py36           40.2.0-1               EasyInstall and python eggs
 i   setuptools-tng-py37           40.2.0-1               EasyInstall and python eggs
```
```
/sw/bin/invoke-py34 test || exit 2
Python version: 3.4.9 (default, Mar 13 2019, 03:57:16) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
...........................................................................................................................................................................................................................................................................................................................................
----------------------------------------------------------------------
Ran 331 tests in 1.921s

OK
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
Destroying test database for alias 'default'...
Name                                        Stmts   Miss Branch BrPart  Cover
-----------------------------------------------------------------------------
localflavor/__init__.py                         1      0      0      0   100%
localflavor/ar/__init__.py                      0      0      0      0   100%
localflavor/ar/ar_provinces.py                  3      0      0      0   100%
localflavor/ar/forms.py                        98      4     38      3    95%
localflavor/at/__init__.py                      0      0      0      0   100%
localflavor/at/at_states.py                     2      0      0      0   100%
localflavor/at/forms.py                        32      1      8      1    95%
localflavor/au/__init__.py                      0      0      0      0   100%
localflavor/au/au_states.py                     1      0      0      0   100%
localflavor/au/forms.py                        43      3     10      3    89%
localflavor/au/models.py                       69      6      6      3    88%
localflavor/au/validators.py                   55      0     20      0   100%
localflavor/be/__init__.py                      0      0      0      0   100%
localflavor/be/be_provinces.py                  2      0      0      0   100%
localflavor/be/be_regions.py                    2      0      0      0   100%
localflavor/be/forms.py                        14      0      0      0   100%
localflavor/bg/__init__.py                      0      0      0      0   100%
localflavor/bg/models.py                       12      0      0      0   100%
localflavor/bg/utils.py                        14      0      4      0   100%
localflavor/bg/validators.py                   49      0     18      0   100%
localflavor/br/__init__.py                      0      0      0      0   100%
localflavor/br/br_states.py                     2      0      0      0   100%
localflavor/br/forms.py                       115      1     48      1    99%
localflavor/br/models.py                       13      0      0      0   100%
localflavor/ca/__init__.py                      0      0      0      0   100%
localflavor/ca/ca_provinces.py                  3      0      0      0   100%
localflavor/ca/forms.py                        54      2     12      0    97%
localflavor/ch/__init__.py                      0      0      0      0   100%
localflavor/ch/ch_states.py                     2      0      0      0   100%
localflavor/ch/forms.py                        67      4     22      5    90%
localflavor/cl/__init__.py                      0      0      0      0   100%
localflavor/cl/cl_regions.py                    2      0      0      0   100%
localflavor/cl/forms.py                        47      2     16      1    95%
localflavor/cn/__init__.py                      0      0      0      0   100%
localflavor/cn/cn_provinces.py                  2      0      0      0   100%
localflavor/cn/forms.py                        54      0     16      0   100%
localflavor/co/__init__.py                      0      0      0      0   100%
localflavor/co/co_departments.py                2      0      0      0   100%
localflavor/co/forms.py                        31      0      8      0   100%
localflavor/cu/__init__.py                      0      0      0      0   100%
localflavor/cu/choices.py                       6      0      0      0   100%
localflavor/cu/forms.py                        53      2      8      0    97%
localflavor/cu/models.py                       44      0      0      0   100%
localflavor/cu/validators.py                   17      2      4      2    81%
localflavor/cz/__init__.py                      0      0      0      0   100%
localflavor/cz/cz_regions.py                    2      0      0      0   100%
localflavor/cz/forms.py                        62      2     22      2    95%
localflavor/de/__init__.py                      0      0      0      0   100%
localflavor/de/de_states.py                     2      0      0      0   100%
localflavor/de/forms.py                        51      2     18      4    91%
localflavor/deprecation.py                      2      0      0      0   100%
localflavor/dk/__init__.py                      0      0      0      0   100%
localflavor/dk/dk_municipalities.py             8      0      0      0   100%
localflavor/dk/dk_postalcodes.py                3      0      0      0   100%
localflavor/dk/forms.py                        14      0      3      0   100%
localflavor/ec/__init__.py                      0      0      0      0   100%
localflavor/ec/ec_provinces.py                  2      0      0      0   100%
localflavor/ec/forms.py                         5      0      0      0   100%
localflavor/ec/models.py                       13      0      0      0   100%
localflavor/ee/__init__.py                      0      0      0      0   100%
localflavor/ee/ee_counties.py                   3      0      0      0   100%
localflavor/ee/forms.py                        60      1     18      1    97%
localflavor/es/__init__.py                      0      0      0      0   100%
localflavor/es/es_provinces.py                  2      0      0      0   100%
localflavor/es/es_regions.py                    2      0      0      0   100%
localflavor/es/forms.py                        79      0     26      0   100%
localflavor/fi/__init__.py                      0      0      0      0   100%
localflavor/fi/fi_municipalities.py             2      0      0      0   100%
localflavor/fi/forms.py                        29      0      6      0   100%
localflavor/fr/__init__.py                      0      0      0      0   100%
localflavor/fr/forms.py                       118      0     44      0   100%
localflavor/fr/fr_department.py                 3      0      2      0   100%
localflavor/fr/fr_region.py                     3      0      0      0   100%
localflavor/fr/models.py                       22      8      0      0    64%
localflavor/gb/__init__.py                      0      0      0      0   100%
localflavor/gb/forms.py                        27      2      4      0    94%
localflavor/gb/gb_regions.py                    8      0      0      0   100%
localflavor/generic/__init__.py                 0      0      0      0   100%
localflavor/generic/checksums.py               25      0     12      0   100%
localflavor/generic/countries/__init__.py       0      0      0      0   100%
localflavor/generic/countries/iso_3166.py       1      0      0      0   100%
localflavor/generic/countries/sepa.py           1      0      0      0   100%
localflavor/generic/forms.py                   50      0     10      0   100%
localflavor/generic/models.py                  41      0      4      0   100%
localflavor/generic/validators.py              97      0     52      0   100%
localflavor/gr/__init__.py                      0      0      0      0   100%
localflavor/gr/forms.py                        62      3     26      3    93%
localflavor/hr/__init__.py                      0      0      0      0   100%
localflavor/hr/forms.py                       104      3     37      3    96%
localflavor/hr/hr_choices.py                    4      0      0      0   100%
localflavor/hu/__init__.py                      0      0      0      0   100%
localflavor/hu/forms.py                         5      0      0      0   100%
localflavor/hu/hu_counties.py                   3      0      0      0   100%
localflavor/id_/__init__.py                     0      0      0      0   100%
localflavor/id_/forms.py                      116      6     47      4    94%
localflavor/id_/id_choices.py                   3      0      0      0   100%
localflavor/ie/__init__.py                      0      0      0      0   100%
localflavor/ie/forms.py                         5      0      0      0   100%
localflavor/ie/ie_counties.py                   2      0      0      0   100%
localflavor/il/__init__.py                      0      0      0      0   100%
localflavor/il/forms.py                        29      0      8      0   100%
localflavor/in_/__init__.py                     0      0      0      0   100%
localflavor/in_/forms.py                       50      2     10      0    97%
localflavor/in_/in_states.py                    2      0      0      0   100%
localflavor/in_/models.py                      13      0      0      0   100%
localflavor/is_/__init__.py                     0      0      0      0   100%
localflavor/is_/forms.py                       29      0      6      0   100%
localflavor/is_/is_postalcodes.py               2      0      0      0   100%
localflavor/it/__init__.py                      0      0      0      0   100%
localflavor/it/forms.py                        50      1      6      0    98%
localflavor/it/it_province.py                   4      0      4      0   100%
localflavor/it/it_region.py                     9      0      8      0   100%
localflavor/it/util.py                         37      2     14      0    96%
localflavor/jp/__init__.py                      0      0      0      0   100%
localflavor/jp/forms.py                        18      0      2      0   100%
localflavor/jp/jp_prefectures.py                3      0      0      0   100%
localflavor/kw/__init__.py                      0      0      0      0   100%
localflavor/kw/forms.py                        47      2     12      3    92%
localflavor/kw/kw_areas.py                      3      0      0      0   100%
localflavor/kw/kw_governorates.py               3      0      0      0   100%
localflavor/lt/__init__.py                      0      0      0      0   100%
localflavor/lt/forms.py                        56      0     14      0   100%
localflavor/lt/lt_choices.py                    4      0      0      0   100%
localflavor/lv/__init__.py                      0      0      0      0   100%
localflavor/lv/forms.py                        45      0     12      0   100%
localflavor/lv/lv_choices.py                    3      0      0      0   100%
localflavor/ma/__init__.py                      0      0      0      0   100%
localflavor/ma/forms.py                        31      0      4      0   100%
localflavor/ma/ma_provinces.py                  4      0      2      0   100%
localflavor/ma/ma_regions.py                    3      0      0      0   100%
localflavor/md/__init__.py                      0      0      0      0   100%
localflavor/md/choices.py                       7      0      0      0   100%
localflavor/md/forms.py                        20      0      0      0   100%
localflavor/md/models.py                       27      0      0      0   100%
localflavor/md/validators.py                   49      0     13      0   100%
localflavor/mk/__init__.py                      0      0      0      0   100%
localflavor/mk/forms.py                        51      1     16      1    97%
localflavor/mk/mk_choices.py                    3      0      0      0   100%
localflavor/mk/models.py                       33      0      0      0   100%
localflavor/mt/__init__.py                      0      0      0      0   100%
localflavor/mt/forms.py                         8      0      0      0   100%
localflavor/mx/__init__.py                      0      0      0      0   100%
localflavor/mx/forms.py                       121      4     44      5    95%
localflavor/mx/models.py                       63      0      0      0   100%
localflavor/mx/mx_states.py                     3      0      0      0   100%
localflavor/nl/__init__.py                      0      0      0      0   100%
localflavor/nl/forms.py                        38      0     10      1    98%
localflavor/nl/models.py                       53      0      4      0   100%
localflavor/nl/nl_provinces.py                  3      0      0      0   100%
localflavor/nl/validators.py                   34      0      8      0   100%
localflavor/no/__init__.py                      0      0      0      0   100%
localflavor/no/forms.py                        87      5     34      6    91%
localflavor/no/no_municipalities.py             2      0      0      0   100%
localflavor/nz/__init__.py                      0      0      0      0   100%
localflavor/nz/forms.py                        40      0      4      0   100%
localflavor/nz/nz_councils.py                   5      0      0      0   100%
localflavor/nz/nz_provinces.py                  4      0      0      0   100%
localflavor/nz/nz_regions.py                    4      0      0      0   100%
localflavor/pe/__init__.py                      0      0      0      0   100%
localflavor/pe/forms.py                        35     11     12      2    55%
localflavor/pe/pe_region.py                     2      0      0      0   100%
localflavor/pk/__init__.py                      0      0      0      0   100%
localflavor/pk/forms.py                        13      0      0      0   100%
localflavor/pk/models.py                       23      0      0      0   100%
localflavor/pk/pk_states.py                     2      0      0      0   100%
localflavor/pl/__init__.py                      0      0      0      0   100%
localflavor/pl/forms.py                       113      0     40      0   100%
localflavor/pl/pl_administrativeunits.py        2      0      0      0   100%
localflavor/pl/pl_voivodeships.py               2      0      0      0   100%
localflavor/pt/__init__.py                      1      0      0      0   100%
localflavor/pt/forms.py                        57      0     22      0   100%
localflavor/pt/pt_regions.py                    3      0      0      0   100%
localflavor/py_/__init__.py                     0      0      0      0   100%
localflavor/py_/forms.py                        8      0      0      0   100%
localflavor/py_/py_department.py                3      0      0      0   100%
localflavor/ro/__init__.py                      0      0      0      0   100%
localflavor/ro/forms.py                        80      4     30      2    95%
localflavor/ro/ro_counties.py                   2      0      0      0   100%
localflavor/ru/__init__.py                      0      0      0      0   100%
localflavor/ru/forms.py                        22      0      0      0   100%
localflavor/ru/ru_regions.py                    3      0      0      0   100%
localflavor/se/__init__.py                      0      0      0      0   100%
localflavor/se/forms.py                        70      0     22      0   100%
localflavor/se/se_counties.py                   3      0      0      0   100%
localflavor/se/utils.py                        42      2     24      3    92%
localflavor/sg/__init__.py                      0      0      0      0   100%
localflavor/sg/forms.py                        35      0     10      0   100%
localflavor/si/__init__.py                      0      0      0      0   100%
localflavor/si/forms.py                        77      0     30      0   100%
localflavor/si/si_postalcodes.py                3      0      2      0   100%
localflavor/sk/__init__.py                      0      0      0      0   100%
localflavor/sk/forms.py                        20      0      2      0   100%
localflavor/sk/sk_districts.py                  2      0      0      0   100%
localflavor/sk/sk_regions.py                    2      0      0      0   100%
localflavor/tn/__init__.py                      0      0      0      0   100%
localflavor/tn/forms.py                         7      0      0      0   100%
localflavor/tn/tn_governorates.py               2      0      0      0   100%
localflavor/tr/__init__.py                      0      0      0      0   100%
localflavor/tr/forms.py                        40      4     20      3    88%
localflavor/tr/tr_provinces.py                  2      0      0      0   100%
localflavor/ua/__init__.py                      0      0      0      0   100%
localflavor/ua/forms.py                        27      0      4      0   100%
localflavor/ua/models.py                       26      0      0      0   100%
localflavor/ua/ua_regions.py                    2      0      0      0   100%
localflavor/us/__init__.py                      0      0      0      0   100%
localflavor/us/forms.py                        55      5     12      1    91%
localflavor/us/models.py                       43      0      0      0   100%
localflavor/us/us_states.py                    14      0      6      0   100%
localflavor/uy/__init__.py                      0      0      0      0   100%
localflavor/uy/forms.py                        26      1      6      1    94%
localflavor/uy/util.py                          7      0      2      0   100%
localflavor/uy/uy_departments.py                2      0      0      0   100%
localflavor/ve/__init__.py                      0      0      0      0   100%
localflavor/ve/forms.py                        10      0      0      0   100%
localflavor/ve/ve_region.py                     3      0      0      0   100%
localflavor/ve/ve_states.py                     3      0      0      0   100%
localflavor/za/__init__.py                      0      0      0      0   100%
localflavor/za/forms.py                        34      0      6      0   100%
localflavor/za/za_provinces.py                  2      0      0      0   100%
-----------------------------------------------------------------------------
TOTAL                                        3982     98   1054     64    97%
phase test: passed
```
Anyway, added and tested with the absolute paths now; shall I open a new PR or commit this directly to master?

**Comment by nieder** [comment]
Fine to commit directly if you trust the syntax of the patch (which should be fine if you've tested the build and it passed).

**Comment by dhomeier** [comment]
Was not allowed to commit directly to master without a review...
```
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: At least 1 approving review is required by reviewers with write access.
To https://github.com/fink/fink-distributions.git
 ! [remote rejected]       master -> master (protected branch hook declined)
error: failed to push some refs to 'https://github.com/fink/fink-distributions.git'
```
so I went through with the PR.

**Pull request merged**